drivers/shared/executor/executor_test.go

```diff
@@ -190,7 +194,7 @@ func TestExecutor_Start_Kill(pt *testing.T) {
 			require := require.New(t)
 			execCmd, allocDir := testExecutorCommand(t)
 			execCmd.Cmd = "/bin/sleep"
-			execCmd.Args = []string{"10 && hello world"}
```

Comment:
Prior to this, the test never test `kill` because the command was wrong.  Note that the launch command is `sleep '10 && hello world'` and `sleep` terminates right away with invalid time interval.  Here, we change it to `sleep 10` so it lives long enough to test killing.